frontend/src/components/Dashboard/SideBar.js

```diff
@@ -0,0 +1,257 @@
+                dataType: 'json',
+                type: 'POST',
+                cache: false,
+                data: JSON.stringify(
+                    stateIndex < states.length
+                        ? {
+                              keywords: [
+                                  'owner=' + this.props.userInfo.user_name,
+                                  '.count',
+                                  'state=' + states[stateIndex],
+                              ],
+                          }
+                        : {
+                              keywords: [
+                                  'owner=' + this.props.userInfo.user_name,
+                                  '.count',
+                                  '.floating',
+                              ],
+                          },
+                ),
+                contentType: 'application/json; charset=utf-8',
+                success: (data) => {
+                    if (stateIndex < states.length) {
+                        bundlesDict[states[stateIndex]] = data.response.result;
+                        fetchBundles(stateIndex + 1, bundlesDict);
+                    } else {
+                        bundlesDict['floating'] = data.response.result;
+                        const bundles: HTMLElement[] = [];
+                        for (let state in bundlesDict) {
+                            // Only show the non-zero items
+                            if (bundlesDict[state] > 0) {
+                                bundles.push(
+                                    <li key={state} className={classes.subheader}>
+                                        <a
+                                            className={classes.subheader}
+                                            href={
+                                                'https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/bundle_state.py'
+                                            }
+                                        >
+                                            {state}
+                                        </a>
+                                        {': ' + bundlesDict[state]}
+                                    </li>,
+                                );
+                            }
+                        }
+                        this.setState({ bundles: bundles });
+                    }
+                },
+                error: (xhr, status, err) => {
+                    console.error(xhr.responseText);
+                },
+            });
+        };
+        // Start to fetch the bundles' count
+        fetchBundles(0, {});
+    }
+
+    render() {
+        const { classes, userInfo } = this.props;
+        if (!userInfo) {
+            return null;
+        }
+        return (
+            <Card className={cx(classes.card)} elevation={0} style={{ height: '100%' }}>
+                <Avatar src={'https://i.pravatar.cc/30'} className={classes.avatar} />
```

Comment:
This link is for testing purpose only right?

Reply:
yes, I think we'd better include the profile image in a separate PR since the current PR is already huge.

Reply:
@yuqijin can we use a more generic placeholder for profile pics? Such as this one (which is also CC-licensed): https://pixabay.com/vectors/blank-profile-picture-mystery-man-973460/

Rather than hard-coding the URL of the image, you can download it to the frontend directory and then just reference it in the code that way.

Reply:
Good advice. Done